### PR TITLE
name townhall 5 correctly.

### PR DESCRIPTION
**Pull request by Raycoms**
# Changes proposed in this pull request:
- Peter forgot the upper case "H".
- And fix problem with stuck builder.
Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/entity/pathfinding/WalkToProxy.java [hunk 1]
```diff
@@ -90,7 +92,8 @@ private boolean takeTheDirectPath(@NotNull BlockPos target, int range, boolean o
      */
     public boolean walkToBlock(@NotNull BlockPos target, int range, boolean onMove)
     {
-        final double distanceToPath = BlockPosUtil.getDistanceSquared(worker.getPosition(), target);
+        final double distanceToPath = worker.getColonyJob() instanceof JobBuilder ?
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/entity/pathfinding/WalkToProxy.java]: MINOR '?' should be on a new line. [rule](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AOperator_Wrap)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by Raycoms** [comment]
@Kostronor ?

**Comment by marvin-bitterlich** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/389/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Pull request merged**